nuxeo-opencmis-impl/src/main/java/org/nuxeo/ecm/core/opencmis/impl/server/NuxeoRepository.java

```diff
@@ -125,6 +126,19 @@
 
     public static final String ELASTICSEARCH_PROP = "org.nuxeo.cmis.elasticsearch";
 
+    /**
+     * Key of the configuration property {@code "nuxeo.opencmis.relaxCmisSpec"}, default to {@code false}. Setting
+     * this property to {@code true} allows users to relax the CMIS specification 1.1 and use customized CMISQL. Please
+     * be aware the risk of doing so. It will potentially cause query parsing error and currently.
+     * <p>
+     * The relax mode does not follow the CMIS specification 1.1, section 2.1.14.2.4.4, where at most one
+     * {@code CONTAINS()} function MUST be included in a single query statement. Such mode only works for CMIS query
+     * having no {@code JOIN} predicate.
+     *
+     * @see https://jira.nuxeo.com/browse/NXP-19858
+     */
+    public static final String RELAX_CMIS_SPEC = "org.nuxeo.cmis.relaxCmisSpec";
```

Comment:
`org.nuxeo.cmis.relaxCmisSpec` is redundant it says "cmis" twice. Use just `org.nuxeo.cmis.relaxSpec`